h2o-r/h2o-package/R/models.R

```diff
@@ -4747,6 +4751,14 @@ h2o.getModelTree <- function(model, tree_number, tree_class = NA) {
     res$predictions <- as.numeric(res$predictions)
   }
   
+  if(is.logical(res$left_cat_split)){
```

Comment:
This must be here in case there are no categorical splits (we do have such tests) - R would assign "logical" type to such lists and it fails on type checks later on.